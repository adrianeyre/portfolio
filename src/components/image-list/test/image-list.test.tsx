import React from 'react';
import { shallow } from 'enzyme';

import ImageList from '../image-list';

describe('Image List', () => {
	it('Should render correctly', () => {
		const imageListComponent = shallow(<ImageList />);

		expect(imageListComponent).toMatchSnapshot();
	});
});
