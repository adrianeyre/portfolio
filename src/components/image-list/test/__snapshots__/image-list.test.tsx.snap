// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Image List Should render correctly 1`] = `ShallowWrapper {}`;
